cobra/mit/access.py

```diff
@@ -125,12 +126,15 @@ def lookupByClass(self, classNames, parentDn=None, propFilter=None):
             dnQuery = DnQuery(parentDn)
             dnQuery.classFilter = classNames
             dnQuery.queryTarget = 'subtree'
-            if propFilter:
-                dnQuery.propFilter = propFilter
+            self.__setQueryParams(dnQuery, kwargs)
             mos = self.query(dnQuery)
         else:
             classQuery = ClassQuery(classNames)
-            if propFilter:
-                classQuery.propFilter = propFilter
+            self.__setQueryParams(classQuery, kwargs)
             mos = self.query(classQuery)
         return mos
+
+        def __setQueryParams(self, query, queryParams):
+            for param, value in queryParams.iteritems():
```

Comment:
iteritems is not Python 3 compatible.  Please change this to use one of these two idioms:

    from future.utils import iteritems
    ...
    for param, value in queryParams.iteritems():

Or:

    from future.utils import viewitems
    ...
    for param, value in queryParams.viewitems():